RestoreSources.props

```diff
@@ -0,0 +1,9 @@
+<Project ToolsVersion="15.0">
+  <PropertyGroup>
+    <RestoreSources Condition="'$(RestoreSources)' == ''">
```

Comment:
Just out of curiosity do you expect RestoreSources to be set anywhere else? For most of our core repo's we have something that looks like https://github.com/dotnet/corefx/blob/master/dir.props#L74.

Reply:
Thank you for the coding tips.